Quotes.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Added file to instantiate Quote component.

Reply:
You can remove this comment